charts/sda-svc/templates/auth-deploy.yaml

```diff
@@ -133,6 +133,8 @@ spec:
           value: {{ .Values.global.auth.jwtAlg }}
         - name: RESIGNJWT
           value: {{ .Values.global.auth.resignJwt | quote }}
+        - name: PUBLICFILE
+          value: {{ .Values.global.c4gh.publicFile | quote }}
```

Comment:
```suggestion
        - name: PUBLICKEY
          value: {{ .Values.global.c4gh.publicKey | quote }}
```

Reply:
I chose to keep the name `publicfile` consistent across all files since there was already an occurrence with that name. I agree it makes more sense to call it `publickey`.

Reply:
An even better name would be `publicKeyPath` since that is what needs to be set.

Reply:
Adding this as a new issue.